build/Targets/RepoToolset/ProjectDefaults.props

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+    <StrongNameKeyId>MicrosoftShared</StrongNameKeyId>
+    <ChecksumAlgorithm>SHA256</ChecksumAlgorithm>
+    <HighEntropyVA>true</HighEntropyVA>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Authors>Microsoft</Authors>
+    <Serviceable>true</Serviceable>
+    <DevelopmentDependency>false</DevelopmentDependency>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=529443</PackageLicenseUrl>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+
+    <!-- Disable the message indicating we are using a preview SDK. That is understood and by design -->
+    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+
+    <!-- By default do not build nuget package for a project. Project may override. -->
+    <IsPackable>false</IsPackable>
+
+    <!--
+      Official build:
+       - Build standalone Portable PDBs to reduce the size of the binaries.
+       - Convert Portable PDBs to Windows PDBs and publish the converted PDBs to Symbol Store to allow WinDBG, 
+         Watson and other tools to find symbol format they understand.
+
+      Jenkins build:
+       - Embed PDBs to make it easier to debug Jenkins crash dumps.
+
+      Developer build:
+       - Build Windows PDBs in order to provide stack traces in xUnit on .NET Framework (4.7.1)
+       - TODO: https://github.com/dotnet/roslyn/issues/25184
+         Use embedded PDBs once .NET Framework 4.7.2 is released and xUnit runner includes configuration switch to enable Portable PDBs
+    -->
+    <DebugType Condition="'$(RoslynDebugType)' == ''">full</DebugType>
```

Comment:
This is the RepoToolset directory but still contains Roslyn specific variables. Is that intended? Seems like we would remove all of this as a part of embracing these files.

Reply:
Yes, it's in transition. I can't do all the needed changes in one PR. Eventually the files in RepoToolset will be deleted and we will just import Arcade. For now I'm trying to get them in a shape that's easy to diff against Arcade targets and props files.